AgWe-CSharp-Client/AgWe-CSharp-Client.csproj

```diff
@@ -78,7 +95,7 @@
       <Generator>MSBuild:Compile</Generator>
       <SubType>Designer</SubType>
     </ApplicationDefinition>
-    <Page Include="UI\MainWindow.xaml">
+    <Page Include="UI\mainwindow.xaml">
```

Comment:
Somehow this got all screwy. This change didn't actually solve my issue but for some reason VS won't let me change it back now.